src/site/asn1.rst

```diff
@@ -34,6 +34,10 @@ You are provided with
     * A *data.txt* file containing some country details
     * A *continent.txt* file containing country and continent information
 
+* :download:`All of this can be downloaded from here<./data/Assignment1.zip>`
+    * This is a compressed IntelliJ project
+    * Just unzip, put it where you want on your computer, and you should be able to open this project through IntelliJ
```

Comment:
From here, the zip looks like a folder with maven style java files laid out in it. I don't think you can just open this from intellij.

Keep in mind, the actual build/deploy step doesn't run on your computer, it runs off of a fresh repo -- there's no intellij project here without it being in this repo. Could add the project, or give instructions for creating a new [project from existing sources](https://www.jetbrains.com/help/idea/import-project-or-module-wizard.html#create-from-sources).

Reply:
Took me longer than I would like to admit to realize the .gitignore had .iml